Assets/MapzenMap.cs

```diff
@@ -21,81 +20,72 @@ public class MapzenMap : MonoBehaviour
     private List<TileTask> pendingTasks = new List<TileTask>();
     private AsyncWorker worker = new AsyncWorker(4);
 
-    void Start()
-    {
-        // Construct the HTTP request
-        for (int x = 0; x < TileRangeX; ++x)
-        {
-            for (int y = 0; y < TileRangeY; ++y)
-            {
-                int tileX = TileX + x;
-                int tileY = TileY + y;
-
-                var url = string.Format("https://tile.mapzen.com/mapzen/vector/v1/all/{0}/{1}/{2}.json?api_key={3}",
-                              TileZ, tileX, tileY, ApiKey);
+    private UnityIO tileIO = new UnityIO();
 
-                Debug.Log("URL request " + url);
+    void fetchNetworkTile(object userData, Response response) {
```

Comment:
What do you think of calling this `onNetworkTileFetched`? At this point we don't fetch the tile, but we just received the data for it.